internal/commands/server/modify.go

```diff
@@ -2,14 +2,13 @@ package server
 
 import (
 	"fmt"
-
```

Comment:
please keep the separation

Reply:
oops, mb! will do. 

Although - we really should figure out a way to let the linter do this for us..